api/src/main/java/uk/gov/dhsc/htbhf/claimant/scheduler/PaymentCycleScheduler.java

```diff
@@ -0,0 +1,46 @@
+package uk.gov.dhsc.htbhf.claimant.scheduler;
+
+import lombok.extern.slf4j.Slf4j;
+import net.javacrumbs.shedlock.core.SchedulerLock;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+import uk.gov.dhsc.htbhf.claimant.repository.ClosingPaymentCycle;
+import uk.gov.dhsc.htbhf.claimant.repository.PaymentCycleRepository;
+
+import java.time.LocalDate;
+import java.util.List;
+
+@Component
+@Slf4j
+public class PaymentCycleScheduler {
```

Comment:
I think some JavaDoc either here or on the constructor (or both) would be useful.